pkgs/racket-doc/scribblings/reference/treelists.scrbl

```diff
@@ -336,13 +336,14 @@ operation takes @math{O(N)} time.
 @defproc[(treelist-find [tl treelist?] [pred (any/c . -> . any/c)]) any/c]{
 
 Checks each element of @racket[tl] with @racket[pred] until the result
-is a true value, and then returns that element. If no such element is
+is a non-false value, and then returns that element. If no such element is
```

Comment:
The convention throughout the documentation is to use `a true value` (as opposed to `#true`) to mean a value other than `#false`.  So, I don't think this should change.